CONTRIBUTING.md

```diff
@@ -0,0 +1,92 @@
+# Contributing to Sheriff
+
+We would love you to contribute to Sheriff.
+As a contributor, here are the guidelines we would like you to follow:
+
+- [Developing](./DEVELOPMENT.md)
+- [Issues and Bugs](#issue)
+- [Feature Requests](#feature)
+- [Coding Rules](#rules)
+
+## <a name="issue"></a> Found a Bug?
+
+If you find a bug in the source code, you can help us by submitting an issue to our [GitHub Repository](https://github.com/softarc-consulting/sheriff).
```

Comment:
Maybe we cann add here a link to the stackblitz example?

Reply:
Good idea